### ISSUE TITLE
m1, macOS, 14.4.1, latest update 3.2.0 not working, especially the brightness.

### ISSUE THREAD

**Issue by xk05** [Wouter01/MediaMate-Releases#54]
when I click to adjust the brightness button can not show the tips.

**Comment by Ender-Wang** [comment]
Right-click the menu bar icon, do you see the Brightness icon greyed out? That's sth I messed up before lmao.

**Comment by xk05** [comment]
> Right-click the menu bar icon, do you see the Brightness icon greyed out? That's sth I messed up before lmao.

nope, when I use the 3.2.0version, and click the brightness icon, it works but does not show the icon on the screen.

**Comment by Wouter01** [comment]
Which event controller mode have you set?
